frontend/packages/dev-console/package.json

```
@@ -21,6 +21,9 @@
       "devconsole": [
         "integration-tests/**/*.scenario.ts"
       ]
+    },
+    "exposedModules": {
+      "icons": "/src/utils/icons.tsx"
```

```suggestion
      "icons": "src/utils/icons.tsx"
```

Can you also update `packages/kubevirt-plugin/package.json` accordingly?

---

Done.